src/components/Upload/Upload.jsx

```diff
@@ -16,6 +17,10 @@ export class Upload extends React.Component {
         this.onChange = this.onChange.bind(this);
     }
 
+    goToLobby() {
+        history.push('/lobby');
```

Comment:
I thought we'd gone with the approach of using the `history` object passed down through the tree?

Reply:
I just copied this function from the leaderboard page, but happy to change it?

Reply:
That's up to you. Another point in a similar area though, is the `UploadButton` class ever being used?